cpp/tests/strings/array_tests.cu

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/column/column_factories.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/strings/sorting.hpp>
+#include <cudf/strings/copying.hpp>
+#include <cudf/types.hpp>
+
+#include <tests/utilities/base_fixture.hpp>
+#include "./utilities.h"
+
+#include <vector>
+#include <gmock/gmock.h>
+
+
+struct ArrayTest : public cudf::test::BaseFixture {};
+
+TEST_F(ArrayTest, Sort)
```

Comment:
ArrayTest doesn't seem to describe these tests. Maybe use Strings_Column_Test or Strings_View_Test or something that better describes what is being tested.